exporters/otlp/common/src/main/java/io/opentelemetry/exporter/otlp/internal/traces/SpanEventMarshaler.java

```diff
@@ -21,25 +21,29 @@
   private final KeyValueMarshaler[] attributeMarshalers;
   private final int droppedAttributesCount;
 
-  static SpanEventMarshaler[] create(List<EventData> events) {
+  static SpanEventMarshaler[] createRepeated(List<EventData> events) {
```

Comment:
Also a NIT: I'd leave the "create" name overloaded - for single and list arguments as IMO it's a cleaner API.

Reply:
This was actually the only place that missed our current naming scheme of createRepeated.

I think I like the different names but we could consider changing all of them in a different PR.